src/rd.h

```diff
@@ -79,13 +79,17 @@
 */
 static RD_INLINE RD_UNUSED void *rd_calloc(size_t num, size_t sz) {
 	void *p = calloc(num, sz);
+#ifndef _AIX  // On AIX, calloc(0) is NULL
```

Comment:
Have you observed a case where it allocs zero bytes?
If so, this will break that code.